### ISSUE TITLE
User usage statistic submission (6)

### ISSUE THREAD

**Issue by Discordanian** [Discordanian/mmatch#93]
Ask customer to let us see their selections for our research. (See Wiki for suggestions)

**Comment by quequegg** [comment]
As with #88, I suggest we defer this until we show more success.  However; I would really like this information in helping us to get the questions right which in turn will help us succeed.

**Comment by alandgri** [comment]
I agree with deferring. This is at odds with the privacy statement. I would like the privacy statement to be really brief and definitive. I.e. "Your answers to the matching questions are never transmitted anywhere, period." If we do it with permission, we have to disclose that which makes the statement more complicated.
